core/domain/platform_parameter_domain.py

```diff
@@ -231,6 +231,10 @@ def from_dict(
             EvaluationContext. The corresponding EvaluationContext domain
             object.
         """
+        # TODO(#15633): Refactor all the places in the codebase where either
```

Comment:
Oh, to be clear, I meant point to the existing issue, so that whoever fixes it can fix this one too. If you create a new issue then people won't know that both issues are related.

Reply:
Done!